machine/machine_core/machine_virtual.py

```diff
@@ -37,10 +38,19 @@
 MEMORY_MB = 1152
 
 
+class Disk(TypedDict):
```

Comment:
The way we use this is actually pretty much exactly `namedtuple`.  Would you mind porting it over?

Reply:
Sure!